doc/whats_new.rst

```diff
@@ -57,6 +57,8 @@ Changelog
 
     - Add interactive plotting of single trials by right clicking on channel name in epochs browser by `Jaakko Leppakangas`_
 
+    - New logos and logo generation script by `Daniel McCloy`_
```

Comment:
I don't feel strongly that it needs to be in there; I added because
@Eric89GXL suggested it. I'll remove later today when I'm not writing from
a cellphone...
On Aug 11, 2015 2:00 AM, "Alexandre Gramfort" <notifications@github.com>
wrote:

> In doc/whats_new.rst
> <https://github.com/mne-tools/mne-python/pull/2395#discussion_r36724537>:
>
> > @@ -57,6 +57,8 @@ Changelog
> >
> >      - Add interactive plotting of single trials by right clicking on channel name in epochs browser by `Jaakko Leppakangas`_
> >
> > +    - New logos and logo generation script by `Daniel McCloy`_
>
> Let's give credits to anyone helping
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2395/files#r36724537>.
>